src/helpers.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const { MODULE, EXPORTS, REQUIRE } = require('./constants');
+
+module.exports = ({ types: t }) => {
```

Comment:
Might be worth adding a comment explaining why we are using a factory function here.